source/B2BApi.AppTests/Functions/EnqueueMessages/EnqueueMessagesOrchestrationTests.cs

```diff
@@ -0,0 +1,188 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Azure.Messaging.ServiceBus;
+using Energinet.DataHub.Core.FunctionApp.TestCommon.ServiceBus.ListenerMock;
+using Energinet.DataHub.Core.FunctionApp.TestCommon.ServiceBus.ResourceProvider;
+using Energinet.DataHub.Core.Messaging.Communication;
+using Energinet.DataHub.EDI.B2BApi.AppTests.DurableTask;
+using Energinet.DataHub.EDI.B2BApi.AppTests.Fixtures;
+using Energinet.DataHub.EnergySupplying.RequestResponse.IntegrationEvents;
+using Energinet.DataHub.Wholesale.Contracts.IntegrationEvents;
+using Energinet.DataHub.Wholesale.Events.Infrastructure.IntegrationEvents;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Google.Protobuf;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Energinet.DataHub.EDI.B2BApi.AppTests.Functions.EnqueueMessages;
+
+[Collection(nameof(B2BApiAppCollectionFixture))]
+public class EnqueueMessagesOrchestrationTests : IAsyncLifetime
+{
+    public EnqueueMessagesOrchestrationTests(
+        B2BApiAppFixture fixture,
+        ITestOutputHelper testOutputHelper)
+    {
+        Fixture = fixture;
+        Fixture.SetTestOutputHelper(testOutputHelper);
+    }
+
+    private B2BApiAppFixture Fixture { get; }
+
+    public Task InitializeAsync()
+    {
+        Fixture.AppHostManager.ClearHostLog();
+
+        // Clear mappings etc. before each test
+        Fixture.ServiceBusListenerMock.ResetMessageHandlersAndReceivedMessages();
+
+        return Task.CompletedTask;
+    }
+
+    public Task DisposeAsync()
+    {
+        Fixture.SetTestOutputHelper(null!);
+
+        return Task.CompletedTask;
+    }
+
+    [Fact]
+    public async Task Given_FeatureFlagIsDisabled_When_CalculationCompletedEventIsSent_Then_OrchestrationIsNeverStarted()
+    {
+        // Arrange
+        Fixture.EnsureAppHostUsesFeatureFlagValue(false);
```

Comment:
```suggestion
        Fixture.EnsureAppHostUsesFeatureFlagValue(false);
```
Is there a way we can ensure the execution order of the tests?

Since the fixture defaults to true

Reply:
No. I have tested it and set the default value according to which test is executed first. We could ensure the order by implementing an test orderer, but I just don't think it is worth it at the moment.